dask_kubernetes/core.py

```diff
@@ -381,6 +392,8 @@ def scale_up(self, n, pods=None, **kwargs):
         --------
         >>> cluster.scale_up(20)  # ask for twenty workers
         """
+        if dask.config.get('kubernetes.count.max') is not None:
+            n = min(n, dask.config.get('kubernetes.count.max'))
```

Comment:
Would it be worth issuing a warning when `n > kubernetes.count.max`? This may lead to surprising behavior when you can't scale beyond a certain threshold because of a parameter you didn't set.